db/migrate/20190711135901_add_fields_to_proceeding_types.rb

```diff
@@ -0,0 +1,9 @@
+class AddFieldsToProceedingTypes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :proceeding_types, :default_level_of_service_id, :integer
+    add_column :proceeding_types, :default_level_of_service_name, :string
```

Comment:
Having a series of methods with a shared prefix or suffix is often a sign that another object is needed, and I think that maybe the case here. I suspect that the id/name relationship is constant (that is, a level of service with id n will always have the same name), and that there are a smaller number of service levels than there are Proceeding types.

I think what is needed here is another object `ServiceLevel`, which has the attributes `service_id` and `name`. Then in `ProceedingType`:

```ruby
belongs_to :default_level_of_service, class_name: 'ServiceLevel', foreign_key: "default_service_level_id"
```
or perhaps
```ruby
belongs_to :default_service_level, class_name: 'ServiceLevel', foreign_key: "default_service_level_id"
```
Otherwise it is likely that inconsistencies between service id and name will creep into the data.

Reply:
yep, on reflection I agree with Rob

Reply:
I agree with Rob too. It crossed my mind to make `level_of_service` a separate object when I was making the original change, but as we currently only have one level of service it didn't seem like a priority at this point.

But you've convinced me it's worth doing, so I've pushed a commit to change it.